Mozilla/Thunderbird.download.recipe

```diff
@@ -20,9 +20,11 @@ combination is offered.</string>
         <string>en_US</string>
         <key>NAME</key>
         <string>Thunderbird</string>
+        <key>DISABLE_CODE_SIGNATURE_VERIFICATION</key>
+        <string>True</string>
```

Comment:
And this should probably be ```<true />``` instead of <string>True</string> since setting it to ```<string>False</string>```  won't enable verification.